.nvmrc

```diff
@@ -1 +1 @@
-10.19.0
+10.23.0
```

Comment:
I think a global version is expected by dmrunner so you may need to update this first